bindings/rust/wasmedge-sys/src/vm.rs

```diff
@@ -430,6 +452,25 @@ impl Vm {
         self.run_function_async(func_name, params)
     }
```

Comment:
Please replace the origin `run_wasm_from_bytes_async` with the new `run_wasm_from_bytes_async2`, but keep using the original API name `run_wasm_from_bytes_async`. Thanks!